addons/sys_signal/fnc_handleSignalReturn.sqf

```diff
@@ -19,26 +19,37 @@ _args params ["_transmitterClass", "_receiverClass"];
 
 if (count _result > 0) then {
     _result params ["_id", "_signal"];
-    private _maxSignal = missionNamespace getVariable [_transmitterClass + "_best_signal", -992];
-    private _currentAntenna = missionNamespace getVariable [_transmitterClass + "_best_ant", ""];
-    if (_id == _currentAntenna || {(_id != _currentAntenna && {_signal > _maxSignal})}) then {
-        missionNamespace setVariable [_transmitterClass + "_best_signal", _signal];
-        missionNamespace setVariable [_transmitterClass + "_best_ant", _id];
+
+    private _bestSignalStr = format ["%1_best_signal", _transmitterClass];
+    private _bestAntStr = format ["%1_best_ant", _transmitterClass];
+
+    private _maxSignal = missionNamespace getVariable [_bestSignalStr , -992];
+    private _currentAntenna = missionNamespace getVariable [_bestAntStr, ""];
+
+    // TODO: Remove before release 2.7.0
+    diag_log format ["ACRE2 Debug: %1_best_signal: %2 --- %1_best_ant: %3", _transmitterClass, _maxSignal, _currentAntenna];
```

Comment:
`TRACE` and enable debug mode? That way we can keep debug things in for possible later use.

Reply:
yes. you are right.